ts/output/common/Wrapper.ts

```diff
@@ -626,9 +626,14 @@ export class CommonWrapper<
    */
   public getBreakNode(bbox: LineBBox): [WW, WW] {
     const [i, j] = bbox.start || [0, 0];
-    if (this.node.isEmbellished) return [this, this.coreMO()] as any as [WW, WW];
-    if (this.node.isToken || !this.childNodes[i]) return [this, null] as any as [WW, WW];
-    return this.childNodes[i].getBreakNode(this.childNodes[i].getLineBBox(j));
+    if (this.node.isEmbellished) {
+      return [this, this.coreMO()] as any as [WW, WW];
+    }
+    const childNodes = (this.childNodes?.[0]?.node?.isInferred ? this.childNodes[0].childNodes : this.childNodes);
```

Comment:
What if `this.childNodes` does not exist? According to the test, this is a possibility. This might throw an error in the next condition.

Reply:
It actually always will exist; I got too aggressive with the `?`.